working-groups/grammar/README.md

```diff
@@ -0,0 +1,34 @@
+# Grammar Working Group
+![working group status: active][status]
+
+The goal of the working group is to satisfy [RFC 1331] and produce a testable,
```

Comment:
In general, I think it would be helpful to have a documentation of the "Roadmap" here. We probably should establish a standard format for this. I've been thinking about a ROADMAP.md file -- but the idea would be to sketch out:

- Here are the phases of the project
- This is the one we are in, and a rough idea of where you are
- This would be updated periodically when we sync up with the lang team and is a good opportunity to announce to the world major phase changes

Reply:
We don't have to add this in this PR, just something I think would be a nice addition.

Reply:
This is a bit hard since our work is rather exploratory but I think it's a good idea to think over on our next wg-grammar meeting. If you could attend @nikomatsakis, that might be helpful.

Reply:
Filed in the agenda for 2019-03-20, https://github.com/rust-lang-nursery/wg-grammar/issues/28.